ariblake/bloom
# download from production theme

### Goal
Initial theme download 

### Testing Instructions
- Visit [Homepage](https://rll52hx7t4p3wge7-31795118219.shopifypreview.com) make sure site builds without error
- Ensure that all files from the production theme have been downloaded

### Code Request Quality Checklist
